project/api/management/commands/import_narratives.py

```diff
@@ -16,52 +16,58 @@ def dms2dd(degrees, minutes, seconds, direction):
             dd *= -1
         return dd
 
+    def parseQID(id_string):
+        id_list = id_string.split(",")
+        parsed_list = map(lambda each:each[1:], id_list)
+        return list(parsed_list)
 
     def handle(self, *args, **options):
         """ Do your work here """
 
-
-
-
-
-    excel_document = openpyxl.load_workbook('geo_events1.xlsx')
+    excel_document = openpyxl.load_workbook('data.xlsx')
     
     for sheet in excel_document:
         all_rows = sheet.rows
         settings = MapSettings(zoom_min = 2.0,zoom_max = 7.5)
         settings.save()
-        narrative = Narrative.objects.filter(title=sheet.title)
+        narrative = Narrative.objects.filter(title="Napoleonic Wars")
```

Comment:
`sheet.title` was better imo

Reply:
This was for quick testing. Forgot to change it back